docs/core/get-started/sdk.md

```diff
@@ -16,24 +24,30 @@ The Java SDK is available via Maven, on Sonatype's [Central Repository](http://c
 </dependencies>
 ```
 
-You can also [download the JAR](../../java/chain-sdk-latest.jar) as a binary.
+**Gradle** users should add the following to `build.gradle`:
 
-## Ruby
+```
+compile 'com.chain:chain-sdk-java:1.0.1'
+```
 
-The Ruby SDK is available [via Rubygems](https://rubygems.org/gems/chain-sdk). Make sure to use the most recent version whose major and minor components (`major.minor.x`) match your version of Chain Core. Ruby 2.1 or greater is required.
+You can also [download the JAR](https://search.maven.org/remotecontent?filepath=com/chain/chain-sdk-java/1.0.1/chain-sdk-java-1.0.1.jar) as a binary.
```

Comment:
I'm in favor of keeping it because there are workflows outside of Maven/Gradle that use the jarfile directly. The fact that the jar is hosted on Maven is incidental. It also moves away from downloading the jarfile from the local core, which means we won't have to bundle it into the docs anymore.

FWIW, I'm kinda just copying the "Download" section of this: http://square.github.io/okhttp/

Lemme know if that addresses your concern.